### ISSUE TITLE
DOSE NOT WORK IN BRAVE 

### ISSUE THREAD

**Issue by NonStop01** [alexwforsythe/code-blocks#225]

**Comment by W4JEW** [comment]
Yes - because typing everything in ALL CAPS gets the attention you want. 

You might as well just close this "issue" if you could even call it that. It's more of a complaint. An "issue" has more context to it.
